app/containers/LoginPrivateKey/LoginPrivateKey.jsx

```diff
@@ -31,27 +47,98 @@ export default class LoginPrivateKey extends React.Component<Props, State> {
             loginWithPrivateKey(wif)
           }}
         >
-          <div className={styles.centeredInput}>
-            <PasswordInput
-              placeholder="Enter your private key here"
-              value={wif}
-              onChange={(e: Object) => this.setState({ wif: e.target.value })}
-              autoFocus
-            />
-          </div>
-          <Button
-            id="loginButton"
-            primary
-            type="submit"
-            className={styles.loginButtonMargin}
-            renderIcon={LoginIcon}
-            disabled={wif.length < 10}
-            shouldCenterButtonLabelText
-          >
-            Login
-          </Button>
+          {scannerActive ? (
+            <div className={styles.scannerContainer}>
+              {/* eslint-disable-next-line */}
+              <video
+                ref={ref => {
+                  this.scanPreviewElement = ref
+                }}
+              />
+            </div>
+          ) : (
+            <div className={styles.centeredInput}>
+              <PasswordInput
+                placeholder="Enter your private key here"
+                value={wif}
+                onChange={(e: Object) => this.setState({ wif: e.target.value })}
+                autoFocus
+              />
+            </div>
+          )}
+          {scannerActive ? (
```

Comment:
You can combine this and the ternary above.

Reply:
👍 good catch @dvdschwrtz